lib/main.dart

```diff
@@ -10,6 +10,7 @@ class TasklyApp extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    
```

Comment:
Why the empty line here? Please refrain from changing LOC in files that are not required. Thanks